backend/priv/repo/migrations/20200409121902_add_official_to_categories.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Til.Repo.Migrations.AddOfficialToCategories do
+  use Ecto.Migration
+
+  def change do
+    alter table(:categories) do
+      add :official, :boolean, null: false, default: true
```

Comment:
probably does not really matter but i think it's better to have default value set to `false`